dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -121,7 +121,14 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
 
         THROW_IF_FAILED(operationalCode);
 
-        winrt::copy_from_abi(channelInfo.channelExpiryTime, &channelExpiryTime);
+        // Need to manually convert from the ABI::DateTime type to the winrt::DateTime type since they have different contracts.
+        FILETIME filetime;
+        LARGE_INTEGER largeTime;
+        largeTime.QuadPart = channelExpiryTime.UniversalTime;
```

Comment:
Is this in UTC? I believe that expiration time is always UTC, but it would be best to check

Reply:
From [docs](https://learn.microsoft.com/en-us/windows/win32/api/minwinbase/ns-minwinbase-filetime): Contains a 64-bit value representing the number of 100-nanosecond intervals since January 1, 1601 (UTC).

So we should be good.